src/transactionsFactories/transferTransactionsFactory.ts

```diff
@@ -81,7 +86,7 @@ export class TransferTransactionsFactory {
         if (this.gasEstimator === undefined) {
             return false;
         }
-        return this.isGasEstimator(this.gasEstimator);
+        return true;
```

Comment:
Function can also be implemented with `return this.gasEstimator !== undefined`.

Reply:
Indeed. Changed!